src/Features/Core/Portable/Diagnostics/AnalyzerHelper.cs

```diff
@@ -44,6 +42,7 @@ internal static partial class AnalyzerHelper
         internal const string WRN_AnalyzerCannotBeCreatedId = "AD1000";
         internal const string WRN_NoAnalyzerInAssemblyId = "AD1001";
         internal const string WRN_UnableToLoadAnalyzerId = "AD1002";
+        internal const string WRN_AnalyzerReferencesNetFrameworkId = "AD1003";
```

Comment:
@mavasani Is just picking the next AD... code like this is OK or does other stuff need to be updated to?

Reply:
So, investigating a bit, the code around this seems to be a bit of a mess.
I thought these special IDE-only ADxxxx IDs and CS and VB ID constants above were removed them long back, but seems like they have lingered around. The comment below seems to indicate the CS and VB IDs above need to be mentioned for live-build deduping: https://github.com/dotnet/roslyn/blob/25f6b6e16dffb08be5cffae4361776e7b6dc71df/src/Features/Core/Portable/Diagnostics/AnalyzerHelper.cs#L27-L29

However, that de-duping work is supposed to be handled by the code here:https://github.com/dotnet/roslyn/blob/cf55f3a58e47298426fa971d3bd9d8857c746c65/src/Compilers/CSharp/Portable/Compilation/CSharpCompilerDiagnosticAnalyzer.cs#L27.

@jasonmalinowski Can you please file a separate tracking issue for cleaning this up and assign it to me?

Reply:
https://github.com/dotnet/roslyn/issues/48448 was filed per @mavasani's request.